appsettings.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
Wait, why is appsettings.json being deleted? I swear I just saw you work with it on your machine...

Reply:
git ignore was not ignoring it

Reply:
We don't want gitignore to ignore it, do we? It has useful stuff in there